n3fit/src/n3fit/model_gen.py

```diff
@@ -130,13 +130,16 @@ def __call__(self, pdf_layer, mask=None):
 
 def observable_generator(
     spec_dict,
+    fitbasis,
+    extern_lhapdf,
     mask_array=None,
     training_data=None,
     validation_data=None,
     invcovmat_tr=None,
     invcovmat_vl=None,
     positivity_initial=1.0,
     integrability=False,
+    n_replicas=1,
```

Comment:
This variable seems unused.
```suggestion
```

Reply:
It should be used by the `Obs_layer` below but not passed correctly  due to it having a default. Now it is also fixed in [84c6ace](https://github.com/NNPDF/nnpdf/pull/1979/commits/84c6ace122b517379ba0c897f66f1a2fab8ed275) 
https://github.com/NNPDF/nnpdf/blob/84c6ace122b517379ba0c897f66f1a2fab8ed275/n3fit/src/n3fit/model_gen.py#L221